enterprise/server/cmd/goinit/BUILD

```diff
@@ -29,6 +29,14 @@ go_library(
 go_binary(
     name = "goinit",
     embed = [":goinit_lib"],
-    pure = "on",
-    static = "on",
+    # "pure" causes the init file to require an ELF interpreter which the kernel
+    # does not have access to if we are bulding statically.
+    pure = select({
+        "//:static": "auto",
+        "//conditions:default": "on",
+    }),
+    # static = select({
+    #    "//:static": "on",
+    #    "//conditions:default": "auto",
+    # }),
```

Comment:
Hmm in case of NOT having `—config=static`, we would still want to produce the original binary with `static = “on”` here right? Or is it not needed at all?

Reply:
It is not needed and will, in fact, cause the firecracker tests to fail.